include/worker.hpp

```diff
@@ -100,6 +100,39 @@ class Worker
                       types::ObjectMap& objectInterfaceMap,
                       const std::string& vpdFilePath);
 
+    /**
+     * @brief An API to Restore and Backup VPD.
+     *
+     * Note: This API works on set of pre-declared keywords to,
```

Comment:
Seems line an incomplete sentence.

Reply:
Updated.

Reply:
I have updated the comment, but its not showing here.